src/test/java/org/openrewrite/java/migrate/lang/var/VarBaseTest.java

```diff
@@ -181,9 +181,12 @@ void inDefinition() {
                   java("""
                     package com.example.app;
 
+                    import java.util.List;
+                    import java.util.ArrayList;
+                    
                     class A {
                       void m() {
-                          List<Object> os = new List();
+                          List<Object> os = new ArrayList<>();
```

Comment:
Good improvement :) Our parser might be a bit too permissive sometimes :)

Reply:
TBH, there is warning "com/example/app/A.java:7: error: List is abstract; cannot be instantiated List<Object> os = new List<>();" but I think that should be an error.